### DISCUSSION TITLE
Subscription to only return store entities that change?

### DISCUSSION THREAD

**Discussion by TheGameKnave** [salesforce/akita#766] [Q&A] [answered]
## I'm submitting a bug (or feature request?)

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[X] Bug report  
[ ] Performance issue
[X] Feature request
[X] Documentation issue or request
[ ] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

When I `selectEntity`, I get a subscription to a particular entity from the store.

When I `selectAll`, I get all the elements from the store.

## Expected behavior

A method should exist to subscribe to only the entites from a store that change.

## What is the motivation / use case for changing the behavior?

I'm maintaining two entity stores; one to hold raw data and one to hold processed data. I'd like to subscribe to the first store to get changes when any entity is updated, so I can process the data and add to the second store.

## Environment

<pre><code>
Angular version: 11.2.7

</code></pre>

**Comment by NetanelBasal** [comment, marked as answer]
https://datorama.github.io/akita/docs/entities/query-entity#entity-actions

**Reply by TheGameKnave** [reply]
Um, whoops! I got to the get methods which aren't (afaict) observables, and stopped reading because I thought it was the end of the observables.
